sematic/ui/packages/common/src/pages/RunDetails/RunTabs.tsx

```diff
@@ -77,6 +81,8 @@ const RunTabs = (props: RunTabsProps) => {
                 <Tab label="Metrics" value="metrics" />
                 <Tab label="Resources" value="ext_res" />
                 <Tab label="Pods" value="pod_lifecycle" />
+                {/* The below are the tabs added by plugins */}
+                { tabs.map((tab, index) => <Tab key={index} label={tab} value={tab} />)}
```

Comment:
Due to a limitation of MUI, using`<Slot>` here does not work (MUI only looks at its immediate children, not the ones beneath `<Slot>`

So I had to use `addTab` to register the addtional tabs to a Context.

Reply:
After adding more plugins to the `PluginsContext`, how will you filter only the run-related plugins here?

Reply:
Good question,

On line 72, I only extracted the field "RunTabs" from PluginsContext, which especially targets this extension point in the "RunTabs.tsx". In the future, other plugin functionalities should occupy a field of PluginContext other than "RunTabs".